libs/parallelism/algorithms/include/hpx/parallel/algorithms/count.hpp

```diff
@@ -528,21 +480,25 @@ namespace hpx {
             )>
         // clang-format on
         friend typename std::iterator_traits<FwdIter>::difference_type
-        tag_invoke(count_t, FwdIter first, FwdIter last, T const& value)
+        tag_fallback_invoke(
+            count_t, FwdIter first, FwdIter last, T const& value)
         {
             static_assert((hpx::traits::is_forward_iterator<FwdIter>::value),
```

Comment:
Could you please change this to `is_input_iterator` while you're looking at this algorithm?